packages/dids/src/methods/did-key.ts

```diff
@@ -86,6 +64,26 @@ export type KeyWithMulticodec = {
  *     verificationMethods: [{ algorithm = 'secp256k1' }]
  *   }
  * });
+ *
+ * // DID Creation with a KMS
+ * const keyManager = new LocalKeyManager();
+ * const did = await DidJwk.create({ keyManager });
+ *
+ * // DID Resolution
+ * const resolutionResult = await DidJwk.resolve({ did: did.uri });
```

Comment:
```suggestion
 * const resolutionResult = await DidKey.resolve({ did: did.uri });
```

Reply:
^ maybe I'm missing something here because how would the unit tests pass on this

Reply:
Great catch -- as always @nitro-neal ! 🙏